pkg/admission/quota/clusterresourcequota/admission.go

```diff
@@ -93,12 +93,12 @@ func (q *clusterQuotaAdmission) Validate(ctx context.Context, a admission.Attrib
 	// to its name. Namespaces are cluster level object that shouldn't go into this plugin or it get blocked listing
 	// the namespace that is just being created.
 	obj := a.GetObject()
-	objMeta, ok := obj.(metav1.ObjectMetaAccessor)
-	if !ok {
-		klog.Warningf("ClusterQuotaAdmission received non-meta object %T", obj)
+	accessor, err := metav1.Accessor(obj)
+	if err != nil {
```

Comment:
maybe we should only `warn` when we fail to get metadata and this is a namespace resource (based on the admission attributes) ?

Reply:
We don't know if this is a namespace resource based on the admission attributes since it only points to a namespace this request was scoped to IIUIC